src/main/kotlin/com/ampnet/headlesscmsservice/service/ContentServiceImpl.kt

```diff
@@ -0,0 +1,39 @@
+package com.ampnet.headlesscmsservice.service
+
+import com.ampnet.headlesscmsservice.exception.ErrorCode
+import com.ampnet.headlesscmsservice.exception.ResourceNotFoundException
+import com.ampnet.headlesscmsservice.persistence.model.Content
+import com.ampnet.headlesscmsservice.persistence.respository.ContentRepository
+import com.ampnet.headlesscmsservice.service.pojo.ContentListResponse
+import com.ampnet.headlesscmsservice.service.pojo.ContentResponse
+import com.ampnet.headlesscmsservice.service.pojo.ContentUpdateServiceRequest
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.UUID
+
+@Service
+class ContentServiceImpl(private val contentRepository: ContentRepository) : ContentService {
+
+    @Transactional(readOnly = true)
+    override fun findByCoop(coop: String, key: String?, languge: String?): ContentListResponse {
+        val texts = contentRepository.findByCoopAndOptionalKeyAndOptionalLang(coop, key, languge)
+        if (texts.isEmpty()) throw ResourceNotFoundException(
+            ErrorCode.CMS_CONTENT_NOT_FOUND, "Text for $key in $languge for $coop not found"
+        )
+        return ContentListResponse(texts.map { ContentResponse(it) })
+    }
+
+    @Transactional
+    override fun updateContent(request: ContentUpdateServiceRequest): ContentResponse {
```

Comment:
Do we support deleting a certain record?
For example, if we have already set some content for coop-key-lang triple, how to delete it? 
We can only update it to the empty string? If yes, we have 2 scenes that are not equal but should be.

Reply:
https://linear.app/ampnet/issue/SD-119/delete-route-for-content-in-cms-service